src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs

```diff
@@ -530,6 +530,42 @@ protected virtual void Dispose(bool disposing)
             _disposed = true;
```

Comment:
One small change if you can get it, since it doesn't allow me to do suggest it. On `StopAsync()` within Dispose, we need to call `GetAwaiter().GetResult()` instead of just `Wait` since it does a bunch of things like unwrapping aggregate exceptions.

Reply:
Done! Thanks!